### PR TITLE
Add silnite v5

### PR DESCRIPTION
**Pull request by netmute**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/silnite.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask "silnite" do
+  version "5"
```
Comment (miccal) [inline Casks/silnite.rb:2]: Please change this to:
```
version "5,2020.08"
```

File: Casks/silnite.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  sha256 "bd4a314918710af6a2eaeccb8dbf26724b5075a178118f33a719161af697eb7c"
+
+  # eclecticlightdotcom.files.wordpress.com was verified as official when first introduced to the cask
+  url "https://eclecticlightdotcom.files.wordpress.com/2020/08/silnite5.zip"
```
Comment (miccal) [inline Casks/silnite.rb:6]: Please change this to:
```
url "https://eclecticlightdotcom.files.wordpress.com/#{version.after_comma.major}/#{version.after_comma.minor}/silnite#{version.before_comma}.zip"
```

File: Casks/silnite.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  name "silnite"
+  desc "Checks EFI firmware and security data file updates"
+  homepage "https://eclecticlight.co/lockrattler-systhist/"
+
```
Comment (miccal) [inline Casks/silnite.rb:10]: Please add:
```
depends_on macos: ">= :el_capitan"
```

File: Casks/silnite.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  pkg "silnite5/silniteInstaller.pkg"
```
Comment (miccal) [inline Casks/silnite.rb:11]: Please change this to:
```
pkg "silnite#{version.before_comma}/silniteInstaller.pkg"
```

**Comment by netmute** [comment]
👍

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**